File: drDAT/analyses/test1/test1.py
#!/usr/bin/python
import subprocess


def test1():
    cmd = "C:\\Users\\Dwight\\AppData\\Local\\Julia-1.1.1\\bin\\julia.exe C:\\Users\\Dwight\\Documents\\Projects\\Dat\\Dev2\\analyses\\test1\\test1.jl"
    p = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True)
    (output, err) = p.communicate()
    p_status = p.wait()
    return '<img src="http://localhost:5000/img/images/fig3.png" alt="plot2">'

File: drDAT/analyses/test1/test1.jl
using Statistics
using PyPlot
using CSV

rd = CSV.read("C:\\Users\\Dwight\\Documents\\Projects\\data\\data1.csv")
#print(rd.Time)
plot(rd.Time, rd.Value);
savefig("C:/Users/Dwight/Documents/Projects/Dat/Dev2/images/fig3.png",format="png")